src/main/java/mekanism/common/util/MekanismUtils.java

```diff
@@ -1493,7 +1489,7 @@ public static boolean hasUsableWrench(EntityPlayer player, BlockPos pos)
 				return true;
 			}
 			
-			if(isCoFHHammer(tool.getItem()) && ((IToolHammer)tool.getItem()).isUsable(tool, player, pos))
+			if(isCoFHHammer(tool.getItem())) // TODO Implement CoFH Hammer && ((IToolHammer)tool.getItem()).isUsable(tool, player, pos))
```

Comment:
Nah, still the same api package id. `Optional` handles the id from package-info like a modid. All they really did was take the RF API out of the main API

https://github.com/CoFH/CoFHCore/tree/1.12/src/main/java/cofh/api/item

Shouldnt really need a dep for it.